src/app/merge-diff/merge-comparsion/merge-comparison.component.scss

```diff
@@ -17,21 +17,36 @@ limitations under the License.
 SPDX-License-Identifier: Apache-2.0
 */
 
-.sourceCol
-{
-        height: 25vh;;
+.sourceCol {
```

Comment:
With the component Sass files, I tend to include a top-level class and have others be children of that. For example, for this component I might have:

```sass
.mdm-merge-comparison {
    .textArea { }

    .sourceCol { }

    //etc
}
```

```html
<div class="mdm-merge-comparison">
    <div class="sourceCol">
        <!-- ... -->
    </div>
</div>
```

I do this so that there are no conflicts between other global classes e.g. there might be another `.spacer` class.

Also, I learned from from frontend developers I used to work with that [BEM naming](http://getbem.com/naming/) of CSS classes is one standard often used which may be useful.